locatr.go

```diff
@@ -184,6 +198,12 @@ func (l *BaseLocatr) getLocatorStr(userReq string) (string, error) {
 		if len(locators) > 0 {
 			validLocator, err := l.getValidLocator(locators)
 			if err == nil {
+				result := locatrResult{
+					LocatrDescription: userReq,
+					CacheHit:          true,
+					Locatr:            validLocator,
+				}
+				l.locatrResults = append(l.locatrResults, result)
 				l.logger.Info(fmt.Sprintf("Cache hit, key: %s, value: %s", userReq, validLocator))
 				return validLocator, nil
 			} else {
```

Comment:
Else statement can be removed since there's an early return, and the two logs can be in the same scope.